tests/resources/technical-records.json

```diff
@@ -26127,8 +26127,7 @@
           "description": "heavy goods vehicle"
         },
         "vehicleConfiguration": "articulated",
-        "vehicleType": "hgv",
-        "vin": "H12220223"
```

Comment:
I was expecting the vin within the tech record to be renamed to historicVin rather than removed?

Reply:
I think it's just clicked. We are no longer amending the dynamo object. historicVin is a 'calculated' field.